src/core/test/fuzzing/src/test/scala/Fuzzing.scala

```diff
@@ -130,7 +133,11 @@ trait SerializationFuzzing[S <: PipelineStage with MLWritable] extends FuzzingMe
       }
       ()
     } finally {
-      FileUtils.forceDelete(new File(path))
+      try {
+        FileUtils.forceDelete(new File(path))
+      } catch {
+        case _: FileNotFoundException =>
```

Comment:
I believe Daniel already added this in PR #210 .. you might want to reconcile.

Reply:
Added this in an earlier commit so no need any more

Reply:
Done (it was added in the previous fuzzing refactor commit)